### DISCUSSION TITLE
Is it possible to send notifications to multiple channels under MS Teams using flux notification controller?

### DISCUSSION THREAD

**Discussion by Naresh240** [fluxcd/flux2#2680] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58024415/165273823-2daa7c62-6b93-495f-a79d-6455bdd0642e.png)

Here I have added one address, is it possible to add multiple addresses?

I have using incoming webhook connector as address. Please help me if any way to add multiple addresses.

Regards,
Naresh

**Comment by somtochiama** [comment, marked as answer]
Each provider uses one address.
To send notifications to multiple addresses, you would have to create multiple provider and alert resource
